src/panels/config/areas/dialog-area-registry-detail.ts

```diff
@@ -93,10 +93,11 @@ class DialogAreaDetail extends LitElement {
               .value=${this._name}
               @input=${this._nameChanged}
               .label=${this.hass.localize("ui.panel.config.areas.editor.name")}
-              .errorMessage=${this.hass.localize(
+              .validationMessage=${this.hass.localize(
                 "ui.panel.config.areas.editor.name_required"
               )}
-              .invalid=${nameInvalid}
+              required
+              autoValidate
```

Comment:
Why do we add `autoValidate` for this one but not for others?

Reply:
I have removed the places where I have added them, since I'm not sure why I have added them in the first place.

Reply:
Added them could be a little nicer, it is the difference between validating while typing and validating on leaving the field.